src/ekore/anomalous_dimensions/unpolarized/time_like/as3.py

```diff
@@ -0,0 +1,832 @@
+"""The unpolarized time-like NNLO Altarelli-Parisi splitting kernels."""
+
+import numba as nb
+import numpy as np
+from numpy import power as npp
+
+from ....harmonics import w1, w2, w3, w4
+from ....harmonics.constants import zeta2, zeta3
+
+# from eko.constants import zeta2, zeta3
+
+
+# from ....harmonics import cache as c
+
+
+@nb.njit(cache=True)
+def gamma_nsp(N, nf, cache, is_singlet=None):
+    r"""Compute the NNLO non-singlet positive anomalous dimension.
+
```

Comment:
Remember to add the reference to the source code / publication in all these functions.